src/validators.js

```diff
@@ -95,3 +95,14 @@ export function validateAlpha(attribute, value) {
 export function validateAlphaNum(attribute, value) {
   return (typeof value === 'string' || typeof value === 'number') && is.alphaNumeric(value)
 }
+
+export function validateRegex(attribute, value, parameters) {
+  this.requireParameterCount(1, parameters, 'regex')
+
+  if (is.not.string(value) && is.not.number(value)) {
+    return false
+  }
+  const regex = parameters[0] ? new RegExp(`${parameters[0]}`) : undefined
```

Comment:
`this.requireParameterCount` will throw error if parameters[0] doesn't exit so this check is useless.

```
const regex = new RegExp(`${parameters[0]}`)
```
should be enough